cirq-ft/cirq_ft/infra/t_complexity_protocol.py

```diff
@@ -81,7 +81,7 @@ def _is_clifford_or_t(stc: Any, fail_quietly: bool) -> Optional[TComplexity]:
     if isinstance(stc, cirq.ClassicallyControlledOperation):
         stc = stc.without_classical_controls()
 
-    if cirq.has_stabilizer_effect(stc):
+    if cirq.num_qubits(stc) <= 2 and cirq.has_stabilizer_effect(stc):
         # Clifford operation.
         return TComplexity(clifford=1)
```

Comment:
I like this change; it also makes reporting the `TComplexity` more accurate since we won't count large cliffords as a single clifford operation.